jetty9-compat-base/src/main/jetty-base/modules/gae.mod

```diff
@@ -39,6 +39,7 @@ jetty.httpConfig.requestHeaderSize=8192
 jetty.httpConfig.responseHeaderSize=8192
 jetty.httpConfig.sendServerVersion=true
 jetty.httpConfig.sendDateHeader=false
+jetty.server.stopTimeout=30000
```

Comment:
There is a default value in the XML of 30s that will be used if the property is not set.

The property is commented out in the jetty9 module because it is following the jetty convention of just documenting with a commented property how to change the default

The property is not commented out in the compat module because it is following the gae-compat convention of being explicit in one place of key values and not relying on jetty defaults.